src/encoded/schemas/chip_alignment_quality_metric.json

```diff
@@ -0,0 +1,159 @@
+{
+    "title": "ChIP-seq alignment quality metric",
+    "description": "Schema for reporting ChIP-seq alignment quality metric",
+    "id": "/profiles/chip_alignment_quality_metric.json",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "type": "object",
+    "required": ["step_run","quality_metric_of", "award", "lab"],
+    "additionalProperties": false,
+    "identifyingProperties": ["uuid"],
+    "mixinProperties": [
+        { "$ref": "mixins.json#/schema_version" },
+        { "$ref": "quality_metric.json#/properties" },
+        { "$ref": "mixins.json#/uuid" },
+        { "$ref": "mixins.json#/submitted"},
+        { "$ref": "mixins.json#/aliases" },
+        { "$ref": "mixins.json#/attribution" },
+        { "$ref": "mixins.json#/standard_status"},
+        { "$ref": "mixins.json#/assay" }
+   ],
+    "properties": {
+        "schema_version": {
+            "default": "7"
+        },
+        "processing_stage": {
+            "title": "Processing stage",
+            "type": "string",
+            "description": "The degree of post-mapping processing to which the statistics apply",
+            "enum": [
+                "filtered",
+                "unfiltered"
+            ]
+        },
+        "diff_chroms": {
+            "title": "# reads with mate mapped to diff chr passing QC",
+            "type": "number",
+            "description": "flagstats: mate mapped different chr (mapQ>=5)"
+        },
+
+        "diff_chroms_qc_failed": {
+            "title": "# reads with mate mapped to diff chr failing QC",
+            "type": "number",
+            "description": "flagstats: mate mapped different chr (mapQ>=5) - qc failed"
+        },
+        "duplicates": {
+            "title": "# reads with duplicates passing QC",
+            "type": "number",
+            "description": "flagstats: duplicates"
+        },
+        "duplicates_qc_failed": {
+            "title": "# reads with duplicates failing QC",
+            "type": "number",
+            "description": "flagstats: duplicates - qc failed"
+        },
+        "mapped": {
+            "title": "# reads mapped passing QC",
+            "type": "number",
+            "description": "flagstats: mapped"
+        },
+        "mapped_pct": {
+            "title": "% reads mapped passing QC",
+            "type": "number",
+            "description": "flagstats: mapped - percent"
+        },
+        "mapped_qc_failed": {
```

Comment:
Please add minimal values to all the numbers you are specifying in the QC objects